package.json

```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.26.0"
```

Comment:
Along with this do the basic setup of using `createBrowserRouter` in main.jsx

https://reactrouter.com/en/main/start/tutorial

Reply:
@rt4914 hi, added `createBrowserRouter` in main.jsx file